doc/cookbook/source/examples/classifier/mkl_binary_class.rst

```diff
@@ -0,0 +1,55 @@
+==================================================
```

Comment:
can you avoid abbreviations in the filenames. Use multiple_kernel_learning_binary_classifier.rst